homeassistant/components/sensor/nederlandse_spoorwegen.py

```diff
@@ -0,0 +1,136 @@
+"""
+Support for Nederlandse Spoorwegen public transport.
+
+For more details on this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.nederlandse_spoorwegen/
+"""
+from datetime import datetime
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['nsapi==2.7.4']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ATTRIBUTION = "Data provided by NS"
+CONF_DEPARTURES = 'departures'
+CONF_FROM = 'from'
+CONF_TO = 'to'
+CONF_VIA = 'via'
+CONF_EMAIL = 'email'
+CONF_PASSWORD = 'password'
+
+ICON = 'mdi:train'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=120)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_EMAIL): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_DEPARTURES): [{
```

Comment:
Please make sure that it's a [list](https://home-assistant.io/developers/development_validation/#lists).